crates/red_knot_python_semantic/resources/mdtest/import/basic.md

```diff
@@ -91,3 +91,16 @@ reveal_type(c.C)  # revealed: Literal[C]
 ```py path=a/b/c.py
 class C: ...
 ```
+
+## Unresolvable submodule imports
+
+```py
+# Topmost component resolvable, submodule not resolvable:
+import a.foo  # error: [unresolved-import] "Cannot resolve import `a.foo`"
+
+# Topmost component unresolvable:
+import b.foo  # error: [unresolved-import] "Cannot resolve import `b.foo`"
```

Comment:
Yeah, there's definitely room for improvement here. I mainly wanted to add a test that made sure that we emitted a diagnostic on both of these (because I nearly proposed a refactor that would have accidentally made that not-the-case, and no tests failed on my bad patch)